### PR TITLE
CI (circleci) configuration (#1)

### PR DESCRIPTION
**Pull request by jupe**
Covers https://github.com/openstf/minicap/issues/199 .

This need to be activated from [circleci app](https://circleci.com/add-projects/gh/openstf) by maintainers. After that CI verify each PR that build still success.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+jobs:
+  build:
+    docker:
+      - image: bitriseio/android-ndk:latest
```
Comment (koral--) [inline .circleci/config.yml]: I think we should use explicit (pinned) version, not the latest and update it manually.
If NDK gets updated in the image then project may stop building.

Comment (jupe) [reply .circleci/config.yml]: I agree. Pinned version is better here.

Comment (jupe) [reply .circleci/config.yml]: done

**Comment by koral--** [comment]
@jupe sorry for the very late response.
This repository will be archived and won't be updated. The successor is https://github.com/DeviceFarmer/minicap.
We decided to go with Semaphore CI there for all the repositories.
Sorry for the inconvenience.

**Pull request closed without merging**